src/table.js

```diff
@@ -239,8 +239,8 @@ export const emptyCell = (cell, schema) => tr => {
     const content = tableNodeTypes(schema).cell.createAndFill().content;
     if (!cell.node.content.eq(content)) {
       tr.replaceWith(
-        cell.pos - 1,
-        cell.pos + cell.node.nodeSize - 2,
+        cell.pos,
+        cell.pos + cell.node.nodeSize - 1,
```

Comment:
Should this be `tr.replaceWith(cell.pos, cell.pos + cell.node.nodeSize)`?

Reply:
or if you're replacing the content -> `tr.replaceWith(cell.start, cell.start + cell.node.nodeSize - 2)`

Reply:
nah doesn't seem to work with above